### PR TITLE
POWEROPS-1837 shop case and shop run notebook and apis

### PR DESCRIPTION
**Pull request by frnhr**
## Description

 * Brought back the `Case` class
    * renamed to `ShopCase` for parity with `ShopRun`
 * Changed `ShopRun.trigger_case()` to work with `ShopCase` objects
    * it used to work with a cased file
       * this is missing information about workflow and additional SHOP files
 * Updated a notebook that demonstrates how to trigger a case manually

## Checklist:
- [x] Tests added/updated.
- [x] Documentation updated.
- [x] Changelog updated in [CHANGELOG.md](https://github.com/cognitedata/power-ops-sdk/blob/master/CHANGELOG.md).
- [x] Version bumped. If triggering a new release is desired, bump the version number in [pyproject.toml](https://github.com/cognitedata/power-ops-sdk/blob/master/pyproject.toml) per [semantic versioning](https://semver.org/).

The notebook:
notebook

### CODE REVIEW

File: cognite/powerops/client/shop/data_classes/case.py [hunk 1]
```diff
@@ -1,21 +1,18 @@
...
+from cognite.powerops.client.shop.data_classes.shop_file import SHOPFileReference, SHOPFileType
 
-class FileRefT(TypedDict):
-    file: str
-    encoding: str
+logger = logging.getLogger(__name__)
 
 
-class Case:
+class ShopCase:
```
Comment (ddonukis) [inline cognite/powerops/client/shop/data_classes/case.py:15]:
```suggestion
class SHOPCase:
```
Just for consistency in the code base. We've been uppercasing SHOP and other abbreviations so far.

Comment (frnhr) [reply cognite/powerops/client/shop/data_classes/case.py:15]: I actually had it like that in the code, but changed to to `ShopCase` to be consistent with `ShopRun`. We have both `SHOP` and `Shop` in the codebase, not ideal...

Comment (frnhr) [reply cognite/powerops/client/shop/data_classes/case.py:15]: Actually, NM! WE have `ShopRun` and `SHOPRun` in the code 😒

**Review by ddonukis — APPROVED** [review]
👍

File: cognite/powerops/client/shop/data_classes/case.py
Comment (ddonukis) [inline cognite/powerops/client/shop/data_classes/case.py:1]: I don't this `Case` being used anywhere. Do we need it? Otherwise we should remove it.

Comment (frnhr) [reply cognite/powerops/client/shop/data_classes/case.py:1]: Not using `Case` any more. We probably want to take a pass of all things in `data_classes` and see what to keep and what to remove. But that's out of scope for now, there is a whole epic for this. This PR introduces `SHOPCase` and only uses that.

**Pull request merged**